README.md

```diff
@@ -1 +1,60 @@
+* The majority of participating reviewers rejects the proposal -> The proposal is rejected. The participating reviewers and the author will consider whether any part part of the proposal can be improved and/or incorporated into a new proposal
+Community contributors are encouraged to present proposals and to share their opinions on any Request For Enhancement (RFE). However, any approval throughout the workflow must be granted by the Narayana team and its optional members.
+
+[^1]: The time window can be adjusted according to individual and team circumstances. However, the Narayana team is required to provide a different time window within 3 days from the date a new proposal is created.
+
+## 1. Proposal and technical document
+
+To ensure a clear connection between proposals (definition in the appendix) and their corresponding tasks, a dedicated folder for each proposal will be created within this repository. The folder name will be the unique JIRA issue associated with the RFE. This JIRA issue should be opened before creating the proposal.
+
+In order to facilitate open discussions and transparent collaboration, a pull request for the technical document (definition in the appendix) must be raised for each proposal. The discussion will thus evolve through reviews and feedback. When the proposal is given the go-ahead by the reviewers, the pull request will be merged.
+
+Once the technical document has been approved and merged, it cannot be modified. If any improvement or modification is needed, the new technical document will have to be reviewed and approved by all reviewers. It is important to highlight that evaluation of modifications that extend an already approved and merged proposal should focus on new additions, unless they have a significant impact on the already approved and merged proposal.
+
+To facilitate the navigation between the pull request and the related JIRA, a hyperlink to the JIRA should be added to the description of the pull request and, vice versa, a link to the pull request should be attached to the JIRA.
+
+## 2. Implementation Document (optional)
+
+An optional implementation-specific document (definition in the appendix) can be submitted at this stage. This document should contain a clear roadmap to ensure a smooth transition from design to implementation. Although this step is optional, it might become necessary when there are disagreements among reviewers. Any time there is a disagreement on the implementation of the technical document, a pull request for an implementation-specific document must be raised and a discussion should be initiated. Once the reviewers agree on a specific implementation, the pull request should be merged and the RFE must be developed based on that document.
```

Comment:
@jmfinelli now I re-read it, maybe it would not but this is the part. I think it's the `should` and `roadmap`.

`This document should contain a clear roadmap to ensure a smooth transition from design to implementation.`

That being said, it is a bit of definition and the sentence before indicates the definition is in the appendix. What do you think to moving this sentence there?

Reply:
Got it :-) I will move `This document should contain a clear roadmap to ensure a smooth transition from design to implementation` to the appendix. Thanks Tom